docs/css-customization.md

```diff
@@ -373,3 +281,236 @@ This is the button to open the Webchat, if you want to modify its color you have
+This is the element holding all Quick Replies, you can change the way they are shown. Like a column.
+```CSS
+[data-cognigy-webchat-root] .webchat-quick-reply-template-replies-container {
+  display: flex;
+  flex-direction: column;
+}
+
+```
+* *webchat-quick-reply-template-reply*  
+This is the style of the single Quick Reply element, all of them will be modified.
+```CSS
+[data-cognigy-webchat-root] .webchat-quick-reply-template-replies-container .webchat-quick-reply-template-reply {
+    animation: "Some animation" ;
+    border-color:#272343;
+    color: #2d334a;
+}
+
+```
+
+### Buttons
+
+* *webchat-buttons-template-root*  
+This class contains the Buttons element.
+```CSS
+[data-cognigy-webchat-root] .webchat-buttons-template-root {
+  border-radius: 0;
+  animation: "Some animation" ;
+}
+
+```
+* *webchat-buttons-template-header*  
+The container and header of the buttons, you can modify the text position and style.
+```CSS
+[data-cognigy-webchat-root] .webchat-buttons-template-header {
+  text-align: center;
+  font-weight: bold;
+  background: #e3f6f5;
+}
+
+```
+* *webchat-buttons-template-button*  
+The class of a single button, you can add cool animations when hovering and styling, this will take effect for all buttons in the list.  
+```CSS
+[data-cognigy-webchat-root] .webchat-buttons-template .webchat-buttons-template-button {
+    background-color: #fffffe;
+    color: #2d334a;
+}
+
+[data-cognigy-webchat-root] .webchat-buttons-template .webchat-buttons-template-button:hover {
+  animation: "Some animation" 1s ease;
+  animation-iteration-count: 1;
+  font-weight: bold;
+}
+
+```
+
+### Image Gallery
+
+* *webchat-carousel-template-root*  
+The card element a Gallery, here you can increase its size for example.
+```CSS
+[data-cognigy-webchat-root] .webchat-carousel-template-root {
+  width: 200px;
+}
+
+```
+* *webchat-carousel-template-content*  
+The content of the card, you can modify it's height and make it look smaller so you could show more content.
+```CSS
+[data-cognigy-webchat-root] .webchat-carousel-template-content {
+  background: #e3f6f5;
+  height: 200px;
+}
+
+```
+* *webchat-carousel-template-title*  
+The title of the card.
+```CSS
+[data-cognigy-webchat-root] .webchat-carousel-template-title {
+    color:blue;
+}
+
+```
+* *webchat-carousel-template-subtitle*  
+The subtitle of the card.
+```CSS
+[data-cognigy-webchat-root] .webchat-carousel-template-subtitle {
+  color:white;
+}
+```
+* *webchat-carousel-template-button*  
+The Button at the bottom of the card, containing a URL.
+```CSS
+[data-cognigy-webchat-root] .webchat-carousel-template-button {
+  color: #272343;
+  background: #ffd803;
+}
+
+[data-cognigy-webchat-root] .webchat-carousel-template-button:hover {
+  background: #cfb000;
+}
+```
+
+### Media
+
+* *webchat-media-template*  
+The media element can be of three forms, Image, Video or Audio. You can modify its container or put some animations on it
```

Comment:
The media element can be of three forms: Image, Video or Audio. You can modify its container or put some animations on it.

Reply:
fixed